src/Trigger/RabbitMQListener.cs

```diff
@@ -50,6 +55,10 @@ internal sealed class RabbitMQListener : IListener, IScaleMonitor<RabbitMQTrigge
             _functionDescriptor = functionDescriptor ?? throw new ArgumentNullException(nameof(functionDescriptor));
             _functionId = functionDescriptor.Id;
             _scaleMonitorDescriptor = new ScaleMonitorDescriptor($"{_functionId}-RabbitMQTrigger-{_queueName}".ToLower());
+            _sampleCount = options.Value.ScaleOptions?.SampleCount ?? 5;
+            _targetQueueLength = options.Value.ScaleOptions?.TargetQueueLength ?? 1000;
+            _prefetchCount = options.Value.PrefetchOptions?.PrefetchCount ?? 30;

```

Comment:
These defaults need to documented. Please open a doc issue and link here to ensure we do not forget updating docs.

Reply:
Also, as @mathewc mentioned in https://github.com/Azure/azure-functions-rabbitmq-extension/issues/66 , it would be good collect data on optimum values that need to be set for prefetch count. Please open a separate issue to run some tests and adjust these values if needed.

Reply:
Opened issues for https://github.com/Azure/azure-functions-rabbitmq-extension/issues/116 and https://github.com/Azure/azure-functions-rabbitmq-extension/issues/117